docs/tutorial.md

```diff
@@ -49,11 +57,32 @@ get "/" do
 end
 ```
 
-## Run your awesome web project.
+All the code should look like this:
+
+```ruby
+require "./crystal_test/*"
+require "kemal"
+
+module AwesomeWebProject
+  get "/" do
+    "Hello World!"
+  end
+end
+```
+
+## 5. Run your awesome web project.
 
 ```
 crystal build --release src/awesome_web_project.cr
 ./awesome_web_project
 ```
```

Comment:
I think this should be mentioned only for `production` mode. In `development` environment this seems a bit tedious. Having run `crystal src/awesome_web_project.cr` works equally fine and should be encouraged for `development` environment.